ndindex/ndindex.py

```diff
@@ -546,6 +547,169 @@ def broadcast_arrays(self):
         """
         return self
 
+def iter_indices(*shapes, skip_axes=(), _debug=False):
+    """
+    Iterate indices for every element of an arrays of shape `shapes`.
+
+    `shapes` should be tuples of shapes, which are broadcast compatible. Each
```

Comment:
Documenting `shapes` being a tuple does make sense technically but its a bit confusing to mention when it comes to star args. My initial thought was that `shapes` was a single arg and not star args.
```suggestion
    `shapes` should be broadcast compatible. Each
```

Reply:
It's actually saying it "should be tuples", not "should be a tuple", but I can see how this can be confusing.